contracts/rewards/base/emission_manager.cairo

```diff
@@ -63,8 +66,13 @@ func set_emission_per_second{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, r
 end
 
 @external
-func set_claimer{}():
-    # TODO
+func set_claimer{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    _user : felt, 
+    _claimer : felt
+):
+    Ownable.assert_only_owner()
```

Comment:
you should move the logic to a file named `emission_manager_library.cairo` and only expose functions in `emission_manager.cairo`. like it's done for `a_token.cairo` and `a_token_library.cairo` here: https://github.com/aave-starknet-project/aave-starknet-core/tree/main/contracts/protocol/tokenization